internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+# TODO check changelog for notes on undocumented fields
+
+# Setting to `true` enables development mode. This setting is not recommended for production deployments. It can be useful for enabling experimental features and collecting debug information.
+Dev = false # Default
```

Comment:
If it's a default, perhaps it's better to leave the whole line commented? WDYT?

Reply:
Hmmm... like to prevent node ops from copying out of this source file and hard coding defaults? I've been thinking of the _markdown_ as the primary presentation, and so this way we get nice syntax highighted blocks (even better when rendered through docs.chain.link IIUC):
![Screenshot from 2022-06-07 12-17-53](https://user-images.githubusercontent.com/1194128/172443822-3f7a6775-fef1-4c50-aa3a-29fba0372d83.png)
The conversion script could easily present things differently, like by trimming the default/example suffixes e.g.:
Default: `InlineField = 'defaultvalue'`
But it would be simplest to keep this toml source of truth formatted the current way - both the script and the tests currently depend it.

Reply:
I am worried that nops are very likely to copy/paste an example toml and tweak it to their needs. If we have all the defaults set as real values, this would give the undesirable outcome that they hardcode all their defaults.

Reply:
Noted on https://app.shortcut.com/chainlinklabs/story/33618/add-config-validate-command